dozer-integrations/dozer-proto3/pom.xml

```diff
@@ -22,15 +22,14 @@
 
     <modelVersion>4.0.0</modelVersion>
     <parent>
+        <artifactId>dozer-integrations</artifactId>
```

Comment:
Can you flip the order, so groupid is above artifact

Reply:
Done.